packages/fuel-indexer-lib/src/graphql/constants.rs

```diff
@@ -61,4 +61,25 @@ lazy_static! {
         "Option<Identity>",
         "Option<Json>",
     ]);
+
+    pub static ref ABI_TYPE_MAP: HashMap<&'static str, &'static str> = HashMap::from_iter([
```

Comment:
Maybe we can get a rustdoc explaining exactly what these are? The name `ABI_TYPE_MAP` is, on its own, pretty generic/uninformative